contentcuration/contentcuration/viewsets/community_library_submission.py

```diff
@@ -232,10 +237,29 @@ def _mark_previous_pending_submissions_as_superseded(self, submission):
             channel_version__lt=submission.channel_version,
         ).update(status=community_library_submission_constants.STATUS_SUPERSEDED)
 
+    def _add_to_community_library(self, submission):
+        country_codes = sorted(country.code for country in submission.countries.all())
+
+        Change.create_change(
+            generate_added_to_community_library_event(
+                key=submission.channel.id,
+                channel_version=submission.channel_version,
+                categories=submission.categories,
+                country_codes=country_codes,
+            ),
+            created_by_id=submission.resolved_by_id,
+        )
+        apply_channel_changes_task.fetch_or_enqueue(
+            submission.resolved_by,
+            channel_id=submission.channel.id,
+        )
+
     @action(
         methods=["post"],
         detail=True,
         serializer_class=CommunityLibrarySubmissionResolveSerializer,
```

Comment:
I did not catch this before, but it seems that if we don't send any countries in this resolve request, the countries that the submission originally had get removed, and therefore we don't have the countries set once they are mapped to the `kolibri_public` models. Could you take a look, please? I suspect it has something to do with the serializer.